bio/seqkit/wrapper.py

```
@@ -37,13 +37,26 @@
 
-shell(
-    "seqkit {snakemake.params.command}"
-    " --threads {snakemake.threads}"
-    " {extra_input}"
-    " {extra_output}"
-    " {extra}"
-    " --out-file {snakemake.output[0]}"
-    " {input}"
-    " {log}"
-)
+if snakemake.params.get("out_bgzip"):
```

Ruff

Undefined name `snakemake`

(F821)